CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - sourcerpm attribute for Rpm unit
+###Added
```

Comment:
1. Added header need not be duplicated.
```suggestion
```

2. It's not valid markdown, there would have to be a space between ### and Added.

Reply:
Do you want project owners to fix up minor issues like this on your behalf at merge time?